cmd/Makefile.am

```diff
@@ -8,7 +8,8 @@ noinst_PROGRAMS =
 noinst_LIBRARIES =
 
 CHECK_CFLAGS = -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes \
-	-Wno-missing-field-initializers -Wno-unused-parameter
+	-Wno-missing-field-initializers -Wno-unused-parameter \
```

Comment:
Can you remove `-Wno-missing-field-initializers`?

Reply:
Indeed, thank you, nice catch! Indeed `-Wmissing-field-initializers`  is already included in `-Wextra` so I removed both `-W{no-,}missing-field-initializers`.